css/media.css

```diff
@@ -0,0 +1,826 @@
+    margin-right: 28px;
+  }
+  .navbar__right-side .navbar__item:nth-child(3) {
+    margin-right: 24px;
+  }
+  .navbar__right-side .navbar__item:last-child {
+    margin-right: 0;
+  }
+  .search-form {
+    width: 158px;
+    margin-top: 6px; /* for PP */
+  }  
+  .search-form__input {
+    width: 140px;
+  }
+  
+  /*market-section */
+  
+  .market-section {
+    padding-top: 32px;
+  }  
+  .banner {
+    padding: 51px 0 0 53px ;
+    min-height: 253px;
+    background-size: cover;
+  }  
+  .banner__title {
+    max-width: 210px;
+    margin-bottom: 9px;
+  }  
+  .banner__link-more::after {
+    margin-left: 15px;
+  }  
+  .market-section__subtitle {
+    margin: 62px 0 22px 0;
+  }
+  .products__product-card {
+    margin: 0;
+  }  
+  .product-card {
+    display: flex;
+    flex-direction: column;
+    width: 208px;
+    padding: 0 9px 0 11px;
+  }
+  .product-card__picture {
+    margin: 0 0 33px 9px;
+  }
+  .product-card__picture img {
+    width: 172px;
+    height: 160px;
+  }
+
+  /* Promo-section */
+
+  .promo-section {
+    padding: 78px 0 95px 0;
+  }
+  .video-block__picture-wrapper {
+    margin-right: 38px;
+  }
+  .video-block__text-wrapper {
+    margin-top: 77px;
+  }
+  .video-block__text {
+    min-width: auto;
+    max-width: auto;
+    margin: 7px 0 15px 0;
+    font-size: 13px;
+    line-height: 16px;
+  }
+  .link-more::after {
+    margin-left: 15px;
+  }
+  .promo-section__subtitle {
+    margin-top: 92px;
+  }
+  .stories-card {
+    max-width: 277px;
+  }
+  .stories-card__pic {
+    max-width: 100%;
+    min-height: 147px;
+  }
+  .stories-card__descr {
+    margin-bottom: 22px;
+    font-size: 13px;
+    line-height: 16px;
+  }
+  .stories-card__link-more::after {
+    content: url("../img/Arrow-black-25.svg");
+    margin-left: 0px;
+  }
+  .gallery__hashtag {
+    margin: 87px 0 8px 0;
+  }
+
+  /* Footer */
+  .footer-navbar__sign-up {
+    margin: 12px 0 35px 0;
+  }
+}
+
+@media screen and (max-width: 950px) {  
+  .header {
+    max-width: 90%;
+    margin: 0 auto;
+  }
+  .products {
+    justify-content: space-around;
+  }
+  .products__product-card:nth-child(3) {
+    display: none;
+  }
+  .stories {
+    justify-content: space-evenly;
+  }
+  .stories-card:last-child {
+    display: none;
+  }
+  .gallery {
+    max-width: 80%;
+    margin: 0 auto;
+  }
+  .gallery__hashtag {
+    margin-right: 40px;
+  }
+}
+
+@media screen and (max-width: 900px) {
+  .header {
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 5px;
+  }
+  .header__logo {
+    display: none;
+  }
+  .header__logo-mobile {
+    display: inline-block;
+    margin-top: 23px;
+  }
+  .navbar {
+    display: none;
+  }
+  .burger {
+    padding: 0;
+    border: none;
+    cursor: pointer;
+    position: relative;
+    display: inline-block;
+    width: 61px;
+    height: 9px;
+    margin-top: 5px;
+    background-color: #000;
+  }
+  .burger::after {
+    content: '';    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654145)